src/kbmod/search/stack_search.h

```diff
@@ -29,6 +29,7 @@ using Point = indexing::Point;
 using Image = search::Image;
 
 class StackSearch {
+    int extract_max_results();
```

Comment:
Maybe rename this to `get_max_results` or `compute_max_results` to avoid any confusion about this somehow extracting the results from the cached list.

Reply:
Good point - renamed